### PR TITLE
JS best practices: review

### PR DESCRIPTION
**Pull request by dewslyse**
This PR is a clone of the to-do list app. In the app, users can:
- add and remove tasks from the page
- update task descriptions
- mark tasks as complete by selecting the corresponding checkbox.
- clear all completed tasks with the click of a button
- clear all tasks from the page

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const addBtn = document.querySelector('.add-btn');
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const todos = Todos.retrieveTask();
+  const todoText = document.querySelector('#add-item').value;
+  const completed = false;
+  const index = todos.length + 1;
+
+  if (todoText.trim() === '') {
+    alert('Please add a task');
```
Comment (tafaramafemba) [inline src/index.js]: Consider manually styling an error message instead of using the window.alert()

**Review by tafaramafemba — CHANGES_REQUESTED** [review]
Well done for the work you have done so far! you are almost there. Please take note of the requested changes. Very soon your project will be ready to merge!

File: src/index.js (hunk 1 above)
Comment (dewslyse) [reply src/index.js]: Thanks you 😀

**Review by tafaramafemba — APPROVED** [review]
Well done on making all the requested changes. There is nothing else left to say but its time to merge!! All the best in the rest of your projects!

**Pull request merged**